src/main/resources/templates/layout.ftl

```diff
@@ -49,7 +49,7 @@
             m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
-    ga('create', 'UA-136887405-1', 'auto');
+    ga('create', '${googleAnalyticsToken}', 'auto');
     ga('send', 'pageview');
   </script>
   <!-- End Google Analytics -->
```

Comment:
Wrap in <#if enableGoogleAnalytics>

Reply:
Will wrap the snippet in common.ftl as well